components/org.wso2.carbon.identity.conditional.auth.functions.common/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/common/utils/ConfigProvider.java

```diff
@@ -39,6 +40,7 @@ public class ConfigProvider {
     private int connectionTimeout;
     private int readTimeout;
     private int connectionRequestTimeout;
+    private int maxRequestAttemptsForAPIEndpointTimeout = 2;
```

Comment:
shall we keep this simple to read as 'requestRetryCount'

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/778ee19b0b9677f6d2cf96e829414272e804de01